index.html

```diff
@@ -0,0 +1,197 @@
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="multipost">
+      <img src="img/multipost.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+
+
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/tonic2.svg" class="postsvg">
+      <h2>Tonic</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/multi2.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="aboutmyself">
+      <h2>About
+        Myself</h2>
+      <p> Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </section>
+
+
+    <section>
+      <ul class="social-icons">
+        <li>
+          <img src="img/twitter icon.svg">
+        </li>
+        <li>
+          <img src="img/Linkedin icon.svg">
+        </li>
+        <li>
+          <img src="img/Mobile icon.svg">
+        </li>
+        <li>
+          <img src="img/git icon.svg">
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li><br>
+      </ul>
```

Comment:
- [ ] Just like I suggested above about the use of a section tag, I'd suggest you implement that in the about section also. :+1: 
- [ ] Following the best practices, each image should have an `alt` attribute, it helps screen readers identify what an image is all about, I'll suggest you update these images and add an alt attribute. :smile: 

> The alt attribute specifies an alternate text for an area, if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).